### PR TITLE
Splits Up Chameleon Kit

### PR DESCRIPTION
**Pull request by matttheficus**
## What Does This PR Do
Chameleon Kit is now cut down a little and broken up for Traitor Uplinks

**Traitors**
Cham Kit: `10TC` - removed Voice Changer + Stamp, added a non-changer Cham Mask
Cham Voice Changer: `10TC`

The net TC cost for everything is the same at `20TC`.

_(Sidenote: If you still want a stamp, its 1TC)_

**Nukies**
Cham Kit is still 30TC for everything in the old kit (Changer + Stamp)

Added a non-voice changer cham mask + added to maints loot.

## Why It's Good For The Game
Allows for Traitors to be more modular with their TC, allowing them to spend some of it on stealth items without breaking the TC bank for builds that might need some firepower. We should be encouraging Antagonists to re-integrate after goals are done to avoid the giga-spiral of fighting and escalation.

This also removes all Class C from the kit, meaning you can buy this and not immediately being a criminal unless you want a mask.

## Testing
Bought some stuff using the uplink.

## Changelog
:cl:
tweak: Split Chameleon Kit and Chameleon Mask from each other.
tweak: Removed Chameleon Stamp from the Chameleon Kit.
tweak: No change to Nuclear Operative Chameleon Kit.
add: Non-Voice Altering Chameleon Mask.
tweak: Reduced the cost of the Chameleon Modulator from 8 > 5.
/:cl:

### CODE REVIEW

**Comment by Fordoxia** [comment]
Also good for contractors that want the kit, since they don't end up with a duplicate voice changer.

**Comment by Henri215** [comment]
Mention that you decreased the Chameleon Voice Modulator Mask TC cost from 8 to 5.

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -403,11 +403,26 @@
...
 	new /obj/item/clothing/head/chameleon(src)
+	new /obj/item/storage/backpack/chameleon(src)
+	new /obj/item/radio/headset/chameleon(src)
+	new /obj/item/pda/chameleon(src)
 	new /obj/item/clothing/mask/chameleon(src)
+
+/obj/item/storage/box/syndie_kit/chameleon/nukie
+	name = "operative's chameleon kit"
+
+/obj/item/storage/box/syndie_kit/chameleon_nuke/populate_contents()
```
Comment (Henri215) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:414]:
```suggestion
/obj/item/storage/box/syndie_kit/chameleon_nuke
	name = "operative's chameleon kit"

/obj/item/storage/box/syndie_kit/chameleon_nuke/populate_contents()
```
Incorrect path

**Review by Henri215 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 2]
```diff
@@ -403,11 +403,26 @@
...
 	new /obj/item/clothing/head/chameleon(src)
+	new /obj/item/storage/backpack/chameleon(src)
+	new /obj/item/radio/headset/chameleon(src)
+	new /obj/item/pda/chameleon(src)
 	new /obj/item/clothing/mask/chameleon(src)
+
+/obj/item/storage/box/syndie_kit/chameleon/nuke
+	name = "operative's chameleon kit"
+
+/obj/item/storage/box/syndie_kit/chameleon_nuke/populate_contents()
```
Comment (Henri215) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:414]: image

**Review by Henri215 — APPROVED** [review]

**Review by lewcc — APPROVED** [review]

**Pull request merged**